pywt/tests/test__pywt.py

```diff
@@ -55,5 +55,14 @@ def test_upcoef_multilevel():
     assert_allclose(a1, a3)
 
 
+def test_wavelet_repr():
+    from pywt import _pywt
+    wavelet = _pywt.Wavelet('sym8')
+
+    repr_wavelet = eval(wavelet.__repr__())
+
+    assert(wavelet.__repr__() == repr_wavelet.__repr__())
```

Comment:
You shouldn't use plain ``assert`` because that gets stripped when byte-compiling with ``-O``. Use ``assert_`` from numpy.testing instead.